### ISSUE TITLE
Let syncs be initiated from the server

### ISSUE THREAD

**Issue by scttnlsn** [scttnlsn/data.io#5]
Syncs might not always originate from the Socket.IO clients.  Provide some sort of server-side API for initiating syncs and having the result propagated to clients.

**Comment by okv** [comment]
Hi @scttnlsn 
Is it currently possible to initiate sync from server?
p.s. Can't find such information at readme or examples.

**Comment by scttnlsn** [comment]
@okv Yeah, it's possible for a client to initiate syncs server-side.  Here's an example: https://github.com/scttnlsn/data.io/blob/master/examples/server/client.js

**Comment by brucou** [comment]
Hi there. I was reviewing the example aforementioned. The few questions came to mind:
- the server do not seem to initiate the sync but rather to send back some data after receiving a sync from the client. Did I get something wrong?
Code here : 
messages.on('sync', function (sync) {
  sync.notify(sync.client);
});
- if I really want to initiate a sync from the server, how do I access the sync object from the messages object?

My use case is to have real-time synchronization between a backbone store on client and its version on the server. When data is updated on server through its own logic (i.e. not client-related), it should emit the modification to all relevant clients. And naturally vice versa, which is the use case that is pretty well documented so far.
Real-time synchro with concurrent accesses is a difficult problem, and such approach with this nice library would naturally be a simple one, but it is good enough for my needs.

**Comment by scttnlsn** [comment]
@brucou Yeah, the example was that of a server-side client.  I don't believe there is currently a convenient way of triggering a sync directly from the server-side code.  Perhaps exposing a `sync` method directly on the server-side resource object would be the way to go.

**Comment by abelantonios8520** [comment]
hola podrias decirme como logearme
